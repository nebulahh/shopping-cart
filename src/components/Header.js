import React from 'react'

function Header({test}) {
  return (
    <div className='fancy-h1' data-testid='h1tag'>{test}</div>
  )
}

export default Header