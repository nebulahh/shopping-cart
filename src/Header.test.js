import React from "react";
import { render, cleanup, screen } from "@testing-library/react";
import '@testing-library/jest-dom'
import Header from './components/Header'

afterEach(cleanup)

test('render home text', () => { 
  const {} = render(<Header test='Hello' />)
  // const text = screen.getByText(/home/i)
  expect().toMatchSnapshot()
})

test('insert text in h1', () => {
  const {getByTestId, getByText} = render(<Header test='Hello'/>)
  
  expect(getByTestId('h1tag')).toHaveTextContent('Hello')
  expect(getByText('Hello')).toHaveClass('f ancy-h1')
})