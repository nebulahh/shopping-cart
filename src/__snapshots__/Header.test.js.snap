// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render home text 1`] = `undefined`;
